packages/nimble-components/src/table-column/duration-text/tests/table-column-duration-text.spec.ts

```diff
@@ -0,0 +1,310 @@
+import { type Fixture, fixture } from '../../../utilities/tests/fixture';
+import type { TableRecord } from '../../../table/types';
+import { TablePageObject } from '../../../table/testing/table.pageobject';
+import { lang, themeProviderTag } from '../../../theme-provider';
+import { parameterizeNamedList } from '../../../utilities/tests/parameterized';
+import { TableColumnDurationTextPageObject } from '../testing/table-column-duration-text.pageobject';
+
+interface SimpleTableRecord extends TableRecord {
+    field?: number | null;
+    anotherField?: number | null;
+}
+
+class ElementReferences {
+    public table!: Table;
+    public column1!: TableColumnDurationText;
+}
+
+describe('TableColumnDurationText', () => {
+    let table: Table<SimpleTableRecord>;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+    let elementReferences: ElementReferences;
+    let tablePageObject: TablePageObject<SimpleTableRecord>;
+    let pageObject: TableColumnDurationTextPageObject<SimpleTableRecord>;
+    let column: TableColumnDurationText;
+
+    // prettier-ignore
+    async function setup(source: ElementReferences): Promise<Fixture<Table<SimpleTableRecord>>> {
+        return fixture<Table<SimpleTableRecord>>(
+            html`<${themeProviderTag} lang="en-US">
+                    <${tableTag} ${ref('table')} style="width: 700px">
+                        <${tableColumnDurationTextTag} ${ref('column1')} field-name="field" group-index="0">
+                            Duration Column 1
+                        </${tableColumnDurationTextTag}>
+                        <${tableColumnDurationTextTag} field-name="anotherField">
+                            Duration Column 2
+                        </${tableColumnDurationTextTag}>
+                    </${tableTag}>
+                </${themeProviderTag}>`,
+            { source }
+        );
+    }
+
+    beforeEach(async () => {
+        elementReferences = new ElementReferences();
+        ({ connect, disconnect } = await setup(elementReferences));
+        table = elementReferences.table;
+        tablePageObject = new TablePageObject<SimpleTableRecord>(table);
+        pageObject = new TableColumnDurationTextPageObject(tablePageObject);
+        await connect();
+        await waitForUpdatesAsync();
+        column = elementReferences.column1;
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('should export its tag', () => {
+        expect(tableColumnDurationTextTag).toBe(
+            'nimble-table-column-duration-text'
+        );
+    });
+
+    it('can construct an element instance', () => {
+        expect(
+            document.createElement('nimble-table-column-duration-text')
+        ).toBeInstanceOf(TableColumnDurationText);
+    });
+
+    it('reports column configuration valid', () => {
+        expect(column.checkValidity()).toBeTrue();
+    });
+
+    describe('displays blank when', () => {
```

Comment:
Could you move many of these tests to be next to `duration-formatter.ts` instead? That way we know the formatter is being tested if someone is using it independently of the column. Then the column tests could just verify things that require the element.